source/administration/config-settings.rst

```diff
@@ -3269,6 +3268,52 @@ Path and filename of the license file on disk. On startup, if Mattermost cannot
 | This feature's ``config.json`` setting is ``"LicenseFileLocation": ""`` with string input.  |
 +---------------------------------------------------------------------------------------------+
 
+TLS Minimum Version
```

Comment:
@crspeller Can you help review these docs related to the new TLS settings https://github.com/mattermost/mattermost-server/pull/9315 You can skip other changes on this PR.

The exact commit with TLS setting docs is here https://github.com/mattermost/docs/pull/2421/commits/55e574b3cc2d93f26053f9ca3eded4ddba838c12